Biotech/Patches/GeneDefs/GeneDefs_Misc.xml

```diff
@@ -44,5 +44,21 @@
 			</statFactors>
 		</value>
 	</Operation>
+	
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/GeneDef[defName="FireResistant"]/damageFactors</xpath>
+		<value>
+			<PrometheumFlame>0.25</PrometheumFlame>
+			<Flame_Secondary>0.25</Flame_Secondary>
+		</value>
+	</Operation>
+	
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/GeneDef[defName="FireWeakness"]/damageFactors</xpath>
+		<value>
+			<PrometheumFlame>4</PrometheumFlame>
```

Comment:
that's... a loooot...

Reply:
It's the same multiplier for vanilla flame. To what valueshould I reduce it?